grails-shell/src/main/groovy/org/grails/cli/gradle/GradleInvoker.groovy

```diff
@@ -58,6 +58,9 @@ class GradleInvoker {
                 arguments << '--stacktrace'
                 arguments << '-Dgrails.full.stacktrace=true'
             }
+            if (commandLine.remainingArgs || commandLine.undeclaredOptions) {
+                arguments << "-Pargs=${commandLine.remainingArgsWithOptionsString}".toString()
```

Comment:
Is this change necessary? Seems requiring all commands that use GradeInvoker to pass -Pargs= is overkill

I think it would be better to change:

https://github.com/grails/grails-core/blob/master/grails-shell/src/main/groovy/org/grails/cli/gradle/commands/GradleTaskCommandAdapter.groovy#L55

To pass the arguments only when invoking the command